src/convertToObject.js

```diff
@@ -5,8 +5,24 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObject = {};
+
+  const rules = sourceString
+    .split(/;(?![^(]*\))/)
+    .filter((rule) => rule.trim() !== '');
```

Comment:
The `.split()` and `.filter()` methods are chained correctly with proper indentation, which is good for readability.